les/server.go

```diff
@@ -201,7 +205,10 @@ func (s *LesServer) Protocols() []p2p.Protocol {
 	}, nil)
 	// Add "les" ENR entries.
 	for i := range ps {
-		ps[i].Attributes = []enr.Entry{&lesEntry{}}
+		ps[i].Attributes = []enr.Entry{&lesEntry{
+			LesVersion: ServerProtocolVersions[len(ServerProtocolVersions)-1],
```

Comment:
The `LesVersion` is not used. Also, it's weird to put the highest supported protocol version in the ENR. Can we remove the LesVersion right now?

Reply:
done